src/docs/sdk/features.mdx

```diff
@@ -396,6 +396,75 @@ The `fingerprints` field should be set to `["$operationName", "$operationType",
 }
 ```
 
+## GraphQL Performance
+
+The GraphQL Performance integration should match the guidelines for [GraphQL Client Integrations](/sdk/features/#graphql-client-integrations) with a few differences:
+
+The transaction's name should be set with the GraphQL operation name, if possible, otherwise fallback to something unique that makes sense, e.g. the canonical name of the actual/generated class.
+
+The transaction's description should be set with the GraphQL operation name, operation type (`query`, `mutation` or `subscription`) and status code, if possible.
+
+The `request.api_target` should be set with `graphql`.
+
+The `request.data` should be set with the raw GraphQL request payload, if possible. This should be guarded by an opt-in flag, e.g. `sendDefaultPii`.
+
+The `contexts.response.data` should be set with the raw GraphQL response payload, only if there were `errors`. This should be guarded by an opt-in flag, e.g. `sendDefaultPii`.
```

Comment:
if I got it right, processing a response can consume the response and make it unavailable afterwards, depending on the underlying impl, correct? should we point this out here as a heads-up?

Reply:
Added a note, thanks.